resp/resp.go

```diff
@@ -179,12 +195,17 @@ func bufferedIntDelim(br *bufio.Reader) (int64, error) {
 }
 
 func readAllAppend(r io.Reader, b []byte) ([]byte, error) {
-	buf := bytes.NewBuffer(b)
+	buf := getBuffer()
 	// a side effect of this is that the given b will be re-allocated if
```

Comment:
I believe this comment is no longer correct, right? B might still get reallocated if its cap is too low, but it doesn't have anything to do with MinRead anymore

Reply:
You're right. I updated the comment in the latest commit.